openlibrary/plugins/openlibrary/js/image_change.js

```diff
@@ -0,0 +1,77 @@
+export function initChangeImageImport() {
```

Comment:
let's call this covers_change.js so it can be linked up to the the HTML template.

please also add a docblock  at the top exploring the purpose of this file and the pages it runs on.
```
/**
 * Explanation of what this file does
 */
```

Reply:
thank you!